src/liballoc/lib.rs

```diff
@@ -159,6 +159,7 @@ pub mod str;
 pub mod string;
 #[cfg(target_has_atomic = "ptr")]
 pub mod sync;
+pub mod task;
```

Comment:
I think this should also be `#[cfg(target_has_atomic = "ptr")]` so that `Arc` is available